content/docs/tech-insights/track-sonarqube/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Track On-prem Sonarqube metrics
+publishedDate: '2023-08-04T15:15:00.0Z'
+description: How to track and visualize a Sonarqube metrics with Roadie
+---
+
+One of the most advantageous steps a platform team can take to support application developers (app devs) is to offer them a reliable and secure platform for enhancing code quality. This is often achieved by incorporating SonarCloud into the development process, enabling teams to identify and rectify code-related issues effectively.
```

Comment:
Let's use this space to highlight the differences between this tutorial and [the SonarCloud one](https://deploy-preview-1100--roadie.netlify.app/docs/tech-insights/track-sonarcloud/). You can link to the SonarCloud one for the background info.

Reply:
Done I think. let me know